Header header

float64 Latitude
float64 Longitude
float32 EllipsoidalHeight
int16 Roll
int16 Pitch
int16 Heading
uint16 PositionRMS
uint16 ZUPTRMS
uint8 HeadingRMS
uint8 Flags
